<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QDialog" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dashboard {
background-color: #2a6496;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color: #f0f0f0; color: #000000; border: 1px solid #cccccc; padding: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QFrame" name="frame_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
background-color: #fff;
border: none;
border-radius: 5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Grade:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_2"/>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Date:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit"/>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Paid Student</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>00</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
background-color: #fff;
border: none;
border-radius: 5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Date:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_3"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>....</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>00</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_7">
        <property name="styleSheet">
         <string notr="true">QFrame {
background-color: #fff;
border: none;
border-radius: 5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Cash</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Date:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_2"/>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>00</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="frame_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
background-color: #fff;
border: none;
border-radius: 5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Grade</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Semester</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>1&amp;2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="frame_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
background-color: #fff;
border: none;
border-radius: 5px;
}

#frame_4 #radioframe{
     border: 1px solid #838383;
     border-radius: 5px;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Grade:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Gender:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="radioframe">
             <property name="styleSheet">
              <string notr="true">#radiframe{
     border: 1px solid #838383;
     border-radius: 5px;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QRadioButton" name="maleradioButton">
                  <property name="text">
                   <string>M</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="femaleradioButton">
                  <property name="text">
                   <string>F</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="load_btn">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* Style the info_frame element */


#frame_2 {
background-color: #fff;
border: none;
border-radius: 7px;
}


QFrame #totalStudent_btn {
background-color: #f0ad4e;

}

QFrame #totalStudent_btn_2 {
background-color: #5cb85c;
}

QFrame #totalStudent_btn_3 {
background-color: #5cb85c;
}

/* Style for function_frame QPushButton */
#frame_2 QPushButton {
    font-size: 14px;
    padding: 5px 10px;
    border: 2px solid #f0f0f0;
    border-radius: 5px;
}


/*Hover Style for buttons insided the function_frame*/
#frame_2 QPushButton:hover{
border-color: #4c96f7;
font-size: 15px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QPushButton" name="totalStudent_btn_2">
             <property name="text">
              <string>Total Man</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalLabel_2">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QPushButton" name="totalStudent_btn_3">
             <property name="text">
              <string>Total Woman</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalLabel_3">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="totalStudent_btn">
             <property name="text">
              <string>Total Student</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame_9">
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color: #fff; color: #000000; border: 1px solid #cccccc; padding: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="result_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/*Style for the result_frame*/

#result_frame{
border-radius: 3px;
background-color: #fff;
}

/*Style border of QTable*/
QTableWidget{
border-radius: 3px;
border: 1px solid #f0f0f0;
}

/*Style for table header section*/
QHeaderView::section{
border: none;
border-bottom: 1px solid #d0c6ff;
text-align: left;
padding: 3px 5px;
}

/*Style for table items*/
QTableWidget::Item{
border-bottom: 1px solid #d0c6ff;
color: #000;
padding-left: 3px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>39</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>120</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_3 #pushButton{
background-color: #c1e2b3;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
